Gems/Atom/RPI/Code/Source/RPI.Public/Image/ImageSystem.cpp

```diff
@@ -347,6 +389,14 @@ namespace AZ
                 Data::AssetId m_assetId;
             };
 
+            // Sync values from desc back to the cvars
+            // Note: we need this code because one of the instance of the cvar might be initialized early than setting registry
+            // due to this issue: https://github.com/o3de/o3de/issues/5537
+            AZ::CVarFixedString commandString = AZ::CVarFixedString::format("r_streamingImagePoolBudgetMb %d", desc.m_systemStreamingImagePoolSize);
+            AZ::Interface<AZ::IConsole>::Get()->PerformCommand(commandString.c_str());
+            commandString = AZ::CVarFixedString::format("r_streamingImageMipBias %d", desc.m_systemStreamingImagePoolMipBias);
```

Comment:
The format specifier here needs to be `"%" PRIu16`: https://en.cppreference.com/w/c/types/integer#Format_macro_constants
```suggestion
            commandString = AZ::CVarFixedString::format("r_streamingImageMipBias %" PRIu16, desc.m_systemStreamingImagePoolMipBias);
```